pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -13,6 +13,7 @@
 , configureFlags ? []
 , description ? ""
 , doCheck ? !isCross && (stdenv.lib.versionOlder "7.4" ghc.version)
+, doBench ? false
```

Comment:
This option suggests to users that the build would run (or compile) the package's benchmarks, but in fact it does not. That feels a little counter-intuitive and misleading.